### ISSUE TITLE
Use admist without offical minecraft launcher

### ISSUE THREAD

**Issue by GalaxyLinus** [toolbox4minecraft/amidst#945]
Hi, I currently have a problem with my original MC Launcher, so I use MultiMC. I tried redirecting Admist to my new .minecraft folder but it doesn't work. Can I somehow add any needed Client files to my broken %appdata%/.minecraft folder so it would work? If yes what files are needed so I can ask my friend to send me them. PS: I only need to load seeds

**Comment by Boobies** [comment]
No, Amidst only supports the official launcher and there is no decent workaround. People have asked for this before but this seems to be the position of the developer on this matter.
